apps/qtmozembed/declarativewebpage.cpp

```diff
@@ -432,40 +402,6 @@ void DeclarativeWebPage::resetViewMargins()
     setMargins(margins);
 }
 
```

Comment:
This doesn't build now...

See DeclarativeWebPage::updateViewMargins() and m_virtualKeyboardMargin. Just change to use virtualKeyboardHeight().